modules/aggregation-layers/src/grid-layer/grid-layer.js

```diff
@@ -334,8 +353,8 @@ export default class GridLayer extends CompositeLayer {
         getFillColor: this._onGetSublayerColor.bind(this),
         getElevation: this._onGetSublayerElevation.bind(this),
         transitions: transitions && {
-          getFillColor: transitions.getColorValue,
-          getElevation: transitions.getElevationValue
+          getFillColor: transitions.getColorValue || transitions.getColorWeight,
+          getElevation: transitions.getElevationValue || transitions.getElevationWeight
```

Comment:
> ```
> getFillColor: transitions.getColorValue || transitions.getColorWeight,
> getElevation: transitions.getElevationValue || transitions.getElevationWeight
> ```
When using new props, color value of cell is defined together by `getColorWeight` and `colorAggregation`.  But given transition key must be prop name just using `getColorWeight`.  Another option is, we can let users use `getColorValue` as key even when `getColorWeight` and `colorAggregation` are specified or use a more generic key as `colorValue` (especially for CompositeLayers) that can be mapped to a sub-layer's attribute/accessor name.